src/blue-theme.scss

```diff
@@ -23,3 +22,11 @@ $primary: $democracy-blue;
     height: 28px;
 }
 
+[class*="toast-display"] {
+    position: absolute;
+    z-index: 1000000;
+}
+
+[class*="toast"] {
+    margin-bottom: 0 !important;
+}
```

Comment:
I haven't checked, but does this .scss file by default ensure that "default" toasts are the same colour as primary?